pkg/environment/environment.go

```diff
@@ -287,3 +288,82 @@ func Get(uuid uuid.UUID) (*Environment, error) {
 		return e, nil
 	}
 }
+
+//TODO add tests
+func IsValid(envID string) (bool, error) {
+	environments, err := GetAll()
+	if err != nil {
+		return false, err
+	}
+	isEnvValid := false
+	for _, e := range environments {
+		if e.Uuid.String() == envID {
+			isEnvValid = true
+			logger.Debug().Msgf("Found environment to export: %s", e.Uuid.String())
+			break
+		}
+	}
+	return isEnvValid, nil
+}
+
+//TODO add tests
+func Export(envID, dstDir string) error {
+	envPath := path.Join(util.UsedEnvironmentDirectory, envID)
+
+	// Final archive name is envID + .zip extension
+	err := archiver.Archive([]string{envPath}, path.Join(dstDir, envID)+".zip")
```

Comment:
```suggestion
	err := archiver.Archive([]string{envPath}, path.Join(dstDir, envID+".zip"))
```
that feels more natural in my opinion